src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -226,7 +228,10 @@ public override async Task<VSCodeAction> ResolveCodeActionsAsync(VSCodeAction co
             return results.FirstOrDefault(c => c != null);
         }
 
-        public override async Task<SemanticTokens> ProvideSemanticTokensAsync(SemanticTokensParams semanticTokensParams, CancellationToken cancellationToken)
+        [Obsolete]
+#pragma warning disable CS0809 // Obsolete member overrides non-obsolete member
```

Comment:
Was this intended? Aka, aren't you warning disabling only because you added the `[Obsolete]` attribute above?

Reply:
It's specifically complaining that  we've marked the override obsolete when the abstract is not. The override is obsolete because it uses `SemanticTokens` (which O# has marked Obsolete due to being proposed) internally.